ext/storage_sources/s3_store/s3_log_system.h

```diff
@@ -82,7 +82,7 @@ class S3LogSystem : public Aws::Utils::Logging::LogSystemInterface {
 
     // Inherited from AWS LogSystemInterface and is not used.
     void
-    Flush()
+    Flush() override
```

Comment:
Should we add the `final` keyword as well?
```suggestion
    Flush() override final
```

Reply:
I thought it was either override or final. Do you mean const override?

Reply:
It is not blocking the PR but it's something I noticed. If we don't intend the inherit from that class and override `Flush`, we could have marked it as `final` too.

Reply:
Done.